src/pipecat/services/elevenlabs.py

```diff
@@ -225,7 +228,9 @@ async def _keepalive_task_handler(self):
         while True:
             try:
                 await asyncio.sleep(10)
-                await self._send_text("")
+                current_time = time.time()
+                if current_time - self._last_message_time >= 10:
+                    await self._send_text("")
```

Comment:
We should update `self._last_message_time` otherwise we could send keepalive continuously.